_posts/2020-08-19-eslint-google-season-of-docs-project-writer.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Introducing the ESLint Google Season of Docs project and technical writer"
+tags:
+  - Google Season of Docs
+  - Technical Writing
+---
+
+# Introducing the ESLint Google Season of Docs project and technical writer
+
+This year, ESLint was accepted into the Google Season of Docs (GSoD) program for the first (thanks, in large part, to our [sponsors](https://opencollective.com/eslint)). The ESLint team has always placed a high value on quality documentation for the project and we came up with a [list of projects](https://github.com/eslint/eslint/issues/13130#issuecomment-614916040) that we thought could improve our documentation. We received more than 20 applications through GSoD, and after a lot of time grading and reviewing applications, we finally ended up with our technical writer!
```

Comment:
Possible missing word-- I took a stab below but disregard if I misunderstood.

```suggestion
This year, ESLint was accepted into the Google Season of Docs (GSoD) program for the first time (thanks, in large part, to our [sponsors](https://opencollective.com/eslint)). The ESLint team has always placed a high value on quality documentation for the project and we came up with a [list of projects](https://github.com/eslint/eslint/issues/13130#issuecomment-614916040) that we thought could improve our documentation. We received more than 20 applications through GSoD, and after a lot of time grading and reviewing applications, we finally ended up with our technical writer!
```

(If the diff doesn't show it, I switched "for the first" to "for the first time")